### PR TITLE
Operator 2.7.0 release notes and documentation update

### PR DESCRIPTION
**Pull request by Nidhi251289**
This contains some improvements along with the releases notes and documentation updates for the new features

### CODE REVIEW

File: content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md [hunk 1]
```diff
@@ -13,6 +13,17 @@ For information on the current status of deployment to Mendix for Private Cloud
...
 ## 2022
 
+### August 23, 2022
+
+#### Mendix Operator v2.7.0{#2.7.0}
+
+* We have changed how TLS trust is configured when building container images; this addresses an issue when OpenShift clusters in Connected mode need to trust custom CA certificates (for example, from a proxy)
+* We have reduced temporary disk usage by about a third when an image is built. The MDA file is now converted directly into a .tar layer instead of being extracted first.
+* The Mendix Operator now restores [ownerReferences](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/) for all managed resources if they are not set.
+* The Mendix Operator now specifies the appProtocol in Kubernetes services to help middleware such as Istio know which [protocol](https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/#explicit-protocol-selection) the app is using and handle traffic as HTTP instead of plan TCP.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md]: Is there a typo here - `plan TCP`?

**Review by MarkvanMents — COMMENTED** [review]
Hi Nidhi @Nidhi251289.
I've made some changes.
There is one outstanding question - I think it is a typo.
Let me know when I can publish.
Thanks for the PR.
Mark

File: content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md (hunk 1 above)
Comment (Nidhi251289) [reply content/en/docs/releasenotes/deployment/mendix-for-private-cloud.md]: it can be changed to just TCP

**Review by MarkvanMents — APPROVED** [review]
Looks good - let me know when the new version is published.

**Pull request merged**